Code/Framework/AzCore/AzCore/std/string/conversions.h

```diff
@@ -384,6 +463,45 @@ namespace AZStd
         return to_wstring(dest, src.c_str(), src.length());
     }
 
+    template<size_t MaxElementCount1>
+    void to_wstring(AZStd::basic_fixed_string<wstring::value_type, MaxElementCount1, wstring::traits_type>& dest, const char* str, size_t strLen = 0)
+    {
+        dest.clear();
+
+        if (strLen == 0)
+        {
+            strLen = strlen(str);
+        }
+
+        if (strLen > 0)
+        {
+            Internal::WCharTPlatformConverter<>::to_wstring(dest, str, str + strLen);
+        }
+    }
+
+    template<size_t MaxElementCount1, size_t MaxElementCount2>
+    void to_wstring(AZStd::basic_fixed_string<wstring::value_type, MaxElementCount1, wstring::traits_type>& dest, const AZStd::basic_fixed_string<string::value_type, MaxElementCount2, string::traits_type>& src)
+    {
+        return to_wstring(dest, src.c_str(), src.length());
+    }
```

Comment:
I thought of a a way to consolidate this function with the `basic_string` function above.
That is to change the input parameter to a string view and therefore only one function is needed

```suggestion
    template<size_t MaxElementCount1>
    void to_wstring(AZStd::basic_fixed_string<wstring::value_type, MaxElementCount1, wstring::traits_type>& dest, const AZStd::basic_string_view<string::value_type, string::traits_type>& src)
    {
        return to_wstring(dest, src.data(), src.length());
    }
```

Reply:
I like it, is also less prone since a lot of calls come from a string_view
I can simplify it even more since `to_wstring` calls always should receive `string_view` and `to_string` should always receive `wstring_view`

Reply:
implemented